tools/utility/dockerise/defaults.py

```diff
@@ -7,6 +7,6 @@
 directory = "NUbots"
 
 
-def is_internal_image(image):
-    split = image.split(":")
-    return len(split) == 2 and split[0] == repository and (split[1] in platform.list() or split[1] == "selected")
+def is_internal_image(tag):
+    split = tag.split(":")
+    return len(split) == 2 and split[0] == image and (split[1] in platform.list() or split[1] == "selected")
```

Comment:
Surely `repository` is more semantically correct here?

Reply:
Nah, in docker you have things split up into 3
`repository/image:thing`
We're checking here that `image` is `nubots` (local image)

Reply:
Fair enough, I dont see any issues then